public/index.html

```diff
@@ -30,6 +32,16 @@
 				height : plateHeight,
 				color  : 'black'
 			});
+
+			this.nodes.push({
+				id : 'ball',
```

Comment:
I think a part of the task could be making ball circle?

Reply:
Yes it'll make the person have to edit app.js & the rendering engine